src/main/java/core/basesyntax/exception/FileValidation.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.exception;
+
+public class FileValidation implements Validator {
+    private static final int TOTAL_LENGTH = 3;
+    private static final int QUANTITY = 2;
+
+    @Override
+    public boolean checkFile(String[] fields, String line) {
+        if (fields.length != TOTAL_LENGTH) {
+            throw new RuntimeException("Length of the line " + fields + " is invalid");
```

Comment:
**fields** is an array and its reference type, it's not informative to print reference in exceptions. Let's print Arrays.toString(fields)